lock/src/main/java/com/auth0/android/lock/views/PanelHolder.java

```diff
@@ -44,10 +48,19 @@ public PanelHolder(Context context) {
         configuration = null;
     }
 
-    public PanelHolder(Context context, Bus lockBus, Configuration configuration) {
+    public PanelHolder(PasswordlessLockActivity context, Bus lockBus, Configuration configuration) {
         super(context);
         this.bus = lockBus;
         this.configuration = configuration;
+        this.usingPasswordless = true;
```

Comment:
Then let's decouple this object into two one for Classic mode and another for Passwordless, this will make the logic in `init()` easier to read